examples/profile-client/src/test/java/com/palantir/example/profile/ProfileStoreTest.java

```diff
@@ -96,6 +99,17 @@ public void testStoreImage() {
     }
 
     @Test
+    public void testGetImageBehaviourWhenNoImage() {
+        final UUID userId = UUID.randomUUID();
+        runWithRetry(store -> {
+            InputStream image = store.getImageForUser(userId);
+            assertThat(image, nullValue());
+            System.out.println("null");
```

Comment:
Won't the database be empty at this point anyway, so any random UUID will return null?